tests/wpt/metadata-css/css-backgrounds-3_dev/html4/attachment-local-clipping-image-6.htm.ini

```diff
@@ -0,0 +1,3 @@
+[attachment-local-clipping-image-6.htm]
```

Comment:
Could you file an issue with details on why this has regressed, and maybe point to it from the ini file?